plugins/modules/cloud/misc/proxmox_disk.py

```diff
@@ -0,0 +1,697 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022, Castor Sky (@castorsky) <csky57@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_disk
+short_description: Management of a disk of a Qemu(KVM) VM in a Proxmox VE cluster.
+version_added: 5.5.0
+description:
+  - Allows you to perform some supported operations on a disk in Qemu(KVM) Virtual Machines in a Proxmox VE cluster.
+author: "Castor Sky (@castorsky) <csky57@gmail.com>"
+options:
+  name:
+    description:
+      - The (unique) name of the VM.
+      - Required only one of I(name) or I(vmid).
+    type: str
+  vmid:
+    description:
+      - The (unique) ID of the VM.
+    type: int
+  disk:
+    description:
+      - The disk key (C(unused[n]), C(ide[n]), C(sata[n]), C(scsi[n]) or C(virtio[n])) you want to operate on.
+    type: str
+    required: True
+  state:
+    description:
+      - Indicates desired state of the disk.
+      - >
+        I(state=present) intended only for creating new disk (with or without replacing existing disk,
+        see I(force_replace) option).
+      - For changing options in existing disk use I(state=updated)
+      - When I(state=detached) and disk is C(unused[n]) it will be left in same state (not removed).
+    type: str
+    choices: ['present', 'updated', 'grown', 'detached', 'moved', 'absent']
+    default: present
+  force_replace:
+    description:
+      - Force replace existing attached disk with the new one leaving old disk unused.
+      - When disk exists and I(force_replace=False) creation will be silently skipped.
+    type: bool
+    default: False
+  storage:
+    description:
+      - The drive's backing storage.
+      - Used primarily when I(state) is C(present) or C(moved).
+    type: str
+  size:
+    description:
+      - For I(state=present) C(size) is desired volume size in GB to allocate.
```

Comment:
Use `I()` for the option name (with or without value), and `C()` for value-only.
```suggestion
      - For I(state=present) I(size) is desired volume size in GB to allocate.
```